components/hab/src/cli/hab/tests.rs

```diff
@@ -5,10 +5,15 @@ use clap::{App,
 use habitat_common::FeatureFlag;
 use std::str;
 
-fn no_feature_flags() -> FeatureFlag { FeatureFlag::empty() }
+fn feature_flags() -> FeatureFlag {
+    let mut f = FeatureFlag::empty();
+    f.insert(FeatureFlag::SERVICE_CONFIG_FILES);
```

Comment:
This looks like `SERVICE_CONFIG_FILES` is somehow a "default" flag... is that intended?

Reply:
This is just for the cli test that compares the `structopt` implementation to the `clap` implementation. The `structopt` version always adds the `bulkload` command regardless of the feature flag. The `clap` version conditionally adds it based on the feature flag with the `maybe_subcommand` function. The `clap` version is the one that is actually used.

This is a limitation of `structopt` it is hard conditionally add subcommands or arguments at runtime because all the code is generated at compile time. The best I have come up with so far is simply hiding the arguments.

Reply:
I meant that the function name `no_feature_flags` before clearly implied there were no feature flags being set, whereas the renamed `feature_flags` is a little less clear. The fact that there's a specific feature flag set in the implementation seems to carry a bit of a "default" flavor.

I think a more intention-revealing name (perhaps along with some documentation comments conveying what you said above) would fix things up nicely.

Reply:
I see, thanks! Addressed in [635ad4](https://github.com/habitat-sh/habitat/pull/7765/commits/635ad4c7934171b337862c0fb808732d561035ad ).

Reply:
Perfect!